src/bundle/Resources/views/themes/admin/url_wildcard/list.html.twig

```diff
@@ -9,6 +10,21 @@
     }
 %}
 
+<section class="container">
+	{{ form_start(form_list, {'attr': {'class': 'ibexa-search-links-form'}}) }}
```

Comment:
Spaces were converted to tabs, compared to the original:
https://github.com/ezsystems/ezplatform-admin-ui/pull/2042/files#diff-fa1753193c6be984045dae4fb4b1858b666d073bab3446abb6d432daaf6af759R11
We use spaces.